test/benchmark.ts

```diff
@@ -87,7 +87,7 @@ describe('Everything', () => {
 
   it('should run everthing using a runner', async () => {
```

Comment:
I think we should change the `it` description as well.

Reply:
Done.